{% extends "layout.html" %}

{% block title %}PiTP{% endblock %}

<!-- Header -->
{% block header2 %}
<br>
<br>
<br>
<br>
<br>
{% endblock %}

<!-- Content -->
{% block content %}

<!-- START THE FEATURETTES -->

<div class="row featurette">
  <div class="col-md-12">
    <h3 class="bs-docs-featurette-title ocustom-droidsans">
      Prospects in Theoretical Physics 2019: Great Problems in Biology for Physicists
    </h3>
    <br>

    <p class="lead">
     PiTP 2019 is titled '<i>Great Problems in Biology for Physicists</i>,' and will cover topics ranging from virology, cancer, and immunology to machine learning and neural networks. Recent technologies have generated enormous amounts of hitherto unseen biological/genetic data. The tools of modern physics have enabled novel approaches to exploration and interrogation of such data, allowing physicists to draw conclusions that will result in leaps of the fundamental understanding of evolution and basic biological processes. Because physicists are now reaching out to explore problems in biology, PiTP 2019 will focus on some of the great questions in biology that are being explored. 
    </p>
 
    <h3 class="bs-docs-featurette-title ocustom-droidsans">
      Some quantitative problems in the evolution and heterogeneity of human cancers
    </h3>
    <br>

    <p class="lead">
     In this lecture we will study how genomics has enabled the characterization of the genetic lesions that are associated with the initiation and development of human cancers. We will focus on four different open questions where quantitative approaches are needed:
    </p>
 
    <ul>
    <div style="font-size: 125%;">
    <li>
    What are the genes that are responsible for the development of human cancers?
    </li>
    <br>
    </div>
    <div style="font-size: 125%;">
    <li>
    Can we reconstruct the mutational history of human cancers?
    </li>
    <br>
    </div>
    <div style="font-size: 125%;">
    <li>
    Can we infer the mutational processes leading to cancer?
    </li>
    <br>
    </div>
    <div style="font-size: 125%;">
    <li>
    What is the role of tumor heterogeneity and the surrounding cells in the progression of cancers?
    </li>
    <br>
    </div>
    </ul>
 
     <p class="lead">
      Below students can find some references and datasets associated with this lecture series.
     </p>
 
  </div>
</div>

<hr>

<div class="row featurette">
  <div class="col-md-12">
    <h3 class="bs-docs-featurette-title ocustom-droidsans"><b>(1)</b> Identifying driver genes in cross-sectional cancer studies</h3>

    <br>

    <p class="lead">
     Papers
    </p>

	<ul>
	<li>
	<div>
	<strong>Mutational heterogeneity in cancer and the search for new cancer-associated genes</strong>
	<br>
	Michael S. Lawrence, Petar Stojanov et al.
	<br>
	<a href="https://www.nature.com/articles/nature12213">Nature. 2013 Jul 11; 499(7457): 214–218.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>The integrated landscape of driver genomic alterations in glioblastoma</strong>
	<br>
	Frattini V. et al.
	<br>
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/23917401">Nature Genetics 2013 Aug 05. doi:10.1038/ng.2734.</a>
	</div>
	</li>
	</ul>


    <p class="lead">
     Datasets
    </p>

	<ul>
	<li>
	<div>
	<a href="https://drive.google.com/open?id=1HSK4x3eTnoOIoeKpbTc1j6oynHpx6Q4r">TCGA_Somatic_Mutations</a>
	</div>
	</li>
	</ul>

  </div>
</div>

<div class="row featurette">
  <div class="col-md-12">
    <h3 class="bs-docs-featurette-title ocustom-droidsans"><b>(2)</b> Reconstructing the evolutionary history of tumors in longitudinal cancer genomic studies</h3>

    <br>

    <p class="lead">
     Papers
    </p>

	<ul>
	<li>
	<div>
	<strong>Clonal evolution mechanisms in NT5C2 mutant-relapsed acute lymphoblastic leukaemia</strong>
	<br>
        Gannie Tzoneva et al.
	<br>
	<a href="https://www.nature.com/articles/nature25186">Nature. Volume 553 (25 January 2018) doi:10.1038/nature25186.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Immune and genomic correlates of response to anti-PD-1 immunotherapy in glioblastoma</strong>
	<br>
        Junfei Zhao et al.
	<br>
	<a href="https://www.nature.com/articles/s41591-019-0349-y">Nat Med. 2019 Feb 11. doi: 10.1038/s41591-019-0349-y.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Clonal evolution of glioblastoma under therapy</strong>
	<br>
	Jiguang Wang et al.
	<br>
	<a href="https://www.nature.com/articles/ng.3590">Nature Genetics volume 48, pages 768–776 (2016).</a>
	</div>
	</li>
	</ul>


    <p class="lead">
     Datasets
    </p>

	<ul>
	<li>
	<div>
	<a href="https://drive.google.com/open?id=1suaVpLQv8Giy8Jmp5codln6n4Nmi5eO8">Glioblastoma variants</a>
	</div>
	</li>
	</ul>

  </div>
</div>

<div class="row featurette">
  <div class="col-md-12">
    <h3 class="bs-docs-featurette-title ocustom-droidsans"><b>(3)</b> Archeology of human cancers: mutational signatures</h3>

    <br>

    <p class="lead">
     Papers
    </p>

	<ul>
	<li>
	<div>
	<strong>Signatures of mutational processes in human cancer</strong>
	<br>
	Ludmil B Alexandrov et al.
	<br>
	<a href="https://www.nature.com/articles/nature12477">Nature volume 500, pages 415–421 (22 August 2013).</a>
	</div>
	</li>
	<li>
	<div>
	<strong>The Repertoire of Mutational Signatures in Human Cancer</strong>
	<br>
	Ludmil B Alexandrov et al.
	<br>
	<a href="https://www.biorxiv.org/content/10.1101/322859v1">doi: https://doi.org/10.1101/322859.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Characterizing Mutational Signatures in Human Cancer Cell Lines Reveals Episodic APOBEC Mutagenesis</strong>
	<br>
	Petljak M et al.
	<br>
	<a href="https://www.cell.com/cell/pdf/S0092-8674(19)30161-8.pdf">Cell, 2019; 176 (6): 1282.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Landscape of somatic mutations in 560 breast cancer whole-genome sequences</strong>
	<br>
	Serena Nik-Zainal et al.
	<br>
	<a href="https://www.nature.com/articles/nature17676">Nature. 2016 May 2; 534(7605): 47–54.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Passenger hotspot mutations in cancer driven by APOBEC3A and mesoscale genomic features</strong>
	<br>
	Rémi Buisson et al.
	<br>
	<a href="https://science.sciencemag.org/content/364/6447/eaaw2872.editor-summary">Science  28 Jun 2019: Vol. 364, Issue 6447, eaaw2872 doi: 10.1126/science.aaw2872.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>A Compendium of Mutational Signatures of Environmental Agents</strong>
	<br>
	Jill Kucab et al.
	<br>
	<a href="https://www.cell.com/cell/pdf/S0092-8674(19)30263-6.pdf">Kucab et al., 2019, Cell 177, 821–836</a>
	</div>
	</li>
	</ul>

    <p class="lead">
     Datasets
    </p>

	<ul>
	<li>
	<div>
	<a href="https://drive.google.com/open?id=1_bpwYR3-U-Rn_X0RLXB01P6OpvJ5FM74">Mutations from 560 WGS breast cancer samples</a>
	</div>
	</li>
	</ul>

  </div>
</div>

<div class="row featurette">
  <div class="col-md-12">
    <h3 class="bs-docs-featurette-title ocustom-droidsans"><b>(4)</b> Studying cancer and stromal heterogeneity using single cell data</h3>

    <br>

    <p class="lead">
     Papers
    </p>

	<ul>
	<li>
	<div>
	<strong>Single-cell topological RNA-seq analysis reveals insights into cellular differentiation and development</strong>
	<br>
	Abbas H Rizvi et al.
	<br>
	<a href="https://www.nature.com/articles/nbt.385://www.nature.com/articles/nbt.3854">Nat Biotechnol. 2017 Jun; 35(6): 551–560.</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Quasi-universality in single-cell sequencing data</strong>
	<br>
	Luis Aparicio et al.
	<br>
	<a href="https://arxiv.org/abs/1810.03602">arXiv preprint arXiv:1810.03602 (2018).</a>
	</div>
	</li>
	<li>
	<div>
	<strong>Single-Cell RNA-Seq Analysis of Infiltrating Neoplastic Cells at the Migrating Front of Human Glioblastoma</strong>
	<br>
	Darmanis S. et al.
	<br>
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/29091775">Cell Rep. 2017 Oct 31;21(5):1399-1410. doi: 10.1016/j.celrep.2017.10.030.</a>
	</div>
	</li>
	</ul>

    <p class="lead">
     Tools
    </p>

	<ul>
	<li>
	<div>
	<strong>Randomly</strong> ( <a href="https://rabadan.c2b2.columbia.edu/html/randomly/">background</a>
	| 
	<a href="https://rabadan.c2b2.columbia.edu/html/randomly/tutorial.html">manual</a>
	|
	<a href="https://github.com/RabadanLab/randomly">Github</a> ) 
	</div>
	</li>
	</ul>


    <p class="lead">
     Datasets
    </p>

	<ul>
	<li>
	<div>
	<a href="https://www.dropbox.com/s/pt1m0uy48iy515q/pbmc_ctrl_annotated.txt.zip?dl=0">PBMC</a>
	</div>
	</li>
	<li>
	<div>
	<a href="https://www.dropbox.com/s/rlyshj58b2wi5wy/GBM.txt.zip?dl=0">GBM</a>
	</div>
	</li>
	</ul>


  </div>
</div>

<hr>

<div class="row featurette">
  <div class="col-md-12">
    <h4 class="bs-docs-featurette-title ocustom-droidsans">Other References</h4>

    <br>

	<ul>
	<li>
	<div>
	<strong>Geometry and topology of genomic data</strong>
	<br>
	A.J. Blumberg and R. Rabadan.
	<br>
	Chapter 64 in J.E. Goodman, J. O'Rourke, and C.D. Toth, editors.
	<br>
	<a href="https://www.crcpress.com/Handbook-of-Discrete-and-Computational-Geometry-Third-Edition/Toth-ORourke-Goodman/p/book/9781498711395">Handbook of Discrete and Computational Geometry, 2nd edition, CRC Press, Boca Raton, FL, 2017, pp. 1735-1773.</a>
	<br>
	<br>
	</div>
	</li>
	<li>
	<div>
	<strong>Spatiotemporal genomic architecture informs precision oncology in glioblastoma</strong>
	<br>
	Lee JK, Wang J, et al. 
	<br>
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/28263318">Nat Genet. 2017 Apr. doi: 10.1038/ng.3806.</a>
	<br>
	<br>
	</div>
	</li>
	<li>
	<div>
	<strong>Topological Data Analysis for Genomics and Evolution: Topology in Biology</strong>
	<br>
	Raul Rabadan and Andrew J. Blumberg.
	<br>
	<a href="https://www.amazon.com/Topological-Data-Analysis-Genomics-Evolution/dp/1107159547">Cambridge University Press. 2019. ISBN-13: 978-1107159549.</a>
	</div>
	</li>
	</ul>

  </div>
</div>

{% endblock %}
